ProQuest.js

```diff
@@ -120,79 +120,96 @@ function fetchEmbeddedPdf(url, item, callback) {
 }
 
 function getSearchResults(doc, checkOnly) {
-	var tabs = doc.getElementsByClassName('tabContent');
 	var root;
-	if (tabs.length) {
-		for (var i = 0; i < tabs.length; i++) {
-			if (tabs[i].offsetHeight) {
-				if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
-				root = tabs[i].getElementsByClassName('resultListContainer')[0];
+	var tabs = doc.getElementsByClassName('tabContent');
+	for (var i = 0; i < tabs.length; i++) {
+		if (tabs[i].offsetHeight) {
+			if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
+			root = tabs[i].getElementsByClassName('resultListContainer')[0];
+			break;
+		}
+	}
+	if (!root) {
+		var elements = doc.getElementsByClassName('resultListContainer');
+		for (var i=0; i<elements.length; i++) {
+			if (elements[i] && elements[i].offsetHeight>0) {
```

Comment:
OK, I see that now. Two things:

(1) we still need the equivalent of `if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;` because the ebrary links are to a different domain and will not work with bookmarklet. It was a simpler check above, but we can do something like `ZU.xpath(elements[i], './/ancestor::div[@id="allResults-content"]').length` now.

(2) Looks like that was my mistake to drop use of `root`. We should be doing `var results = root.querySelectorAll('.resultTitle, .previewTitle')` and then revert back to what we had previously below.